### PR TITLE
Story CORE-591 | Implement benchmarking dapp out of Inspr Stack

### PR DESCRIPTION
**Pull request by LucasStarlingdePaulaSalles**
# Description

Kind: Story

Section: Examples

Summary: An example of write-read kafka dapps without the use of Inspr

## Changelog
- feature:
  - implemented k8s job for creating "kafka-test" topic on kafka
  - implemented k8s job for deleting "kafka-test" topic on kafka
  - implemented k8s deployment for app that sends messages non stop on "kafka-test" topic
  - implemented k8s deployment for app that reads messages non stop from "kafka-test" topic
  - created Makefile and dockerfiles to compile images 
## Pay attention to

> Important parts of the code that require special attention

### CODE REVIEW

**Review by lucasABLima — APPROVED** [review]

File: examples/kafka_standalone/reader/reader.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/insprlabs/inspr/sidecar/kafka:build AS build
+WORKDIR /app
+COPY go.mod go.mod
+RUN go mod download
+COPY . .
+# RUN go build -o main examples/kafka_standalone/topic/create.go
```
Comment (luiz-couto) [inline examples/kafka_standalone/reader/reader.Dockerfile]: Is this comment needed?

File: examples/kafka_standalone/topic/create/create.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/insprlabs/inspr/sidecar/kafka:build AS build
+WORKDIR /app
+COPY go.mod go.mod
+RUN go mod download
+COPY . .
+# RUN go build -o main examples/kafka_standalone/topic/create.go
```
Comment (luiz-couto) [inline examples/kafka_standalone/topic/create/create.Dockerfile]: And this one?

File: examples/kafka_standalone/topic/delete/delete.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/insprlabs/inspr/sidecar/kafka:build AS build
+WORKDIR /app
+COPY go.mod go.mod
+RUN go mod download
+COPY . .
+# RUN go build -o main examples/kafka_standalone/topic/create.go
```
Comment (luiz-couto) [inline examples/kafka_standalone/topic/delete/delete.Dockerfile]: And this?

File: examples/kafka_standalone/writer/writer.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+				} else {
+					fmt.Printf("Successfully produced record to topic %s partition [%d] @ offset %v\n",
+						*ev.TopicPartition.Topic, ev.TopicPartition.Partition, ev.TopicPartition.Offset)
+				}
+			}
+		}
+	}()
+	time.Sleep(5 * time.Second)
+
+	// delivery_chan := make(chan kafka.Event, 10000)
```
Comment (luiz-couto) [inline examples/kafka_standalone/writer/writer.go]: This one too

**Review by luiz-couto — CHANGES_REQUESTED** [review]
Just remove unnecessary comments

**Review by luiz-couto — APPROVED** [review]

**Pull request merged**